lib/js/jsconsole.nim

```diff
@@ -67,4 +69,15 @@ proc timeLog*(console: Console, label = "".cstring) {.importcpp.}
 proc table*(console: Console) {.importcpp, varargs.}
   ## https://developer.mozilla.org/docs/Web/API/Console/table
 
+since (1, 5):
```

Comment:
this doesn't work, you're hitting A1 https://github.com/nim-lang/Nim/issues/13943
```nim
import jsconsole
proc main()=
  var a = 2
  console.doAssert 1 == a
  a = 3
  console.doAssert 1 == a
main()
```

ReferenceError: a is not defined
(look at the codegen)

this works:
```nim
{.emit: ["console.assert(", assertion, ");"].}
```

Reply:
But this hardcodes `true` or `false` at compile-time instead of emitting the actual expression. :thinking:

Reply:
why is that a problem?